components/backline/x86_64-linux-kernel2/plan.sh

```diff
@@ -0,0 +1,23 @@
+pkg_name=hab-backline
+pkg_origin=core
+pkg_version=$(cat "$SRC_PATH/../../VERSION")
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('Apache-2.0')
+pkg_build_deps=()
```

Comment:
Technically, I don't think you do. Convention in core-plans has been to leave it so that it's explicit in the plan file that there are no build dependencies.